src/display-buffer.coffee

```diff
@@ -1174,7 +1174,7 @@ class DisplayBuffer extends Model
 
   destroyed: ->
     fold.destroy() for markerId, fold of @foldsByMarkerId
-    marker.disposables.dispose() for id, marker of @markers
```

Comment:
@nathansobo this used to be `unsubscribe()` instead calling dispose directly before the 1.0 API changes, do you happen to remember why that was and why it didn't always call `destroy()` on the marker?